fmt/README.md

```diff
@@ -17,47 +17,47 @@ below.
 
 This is very much a work-in-progress. I'm actively soliciting feedback.
 
-- What useful features are available in other languages apart from Golang and C?
+-   What useful features are available in other languages apart from Golang and C?
```

Comment:
Please undo all of the style only changes like this. It make the diff bigger and PRs more difficult to review.

Reply:
done